src/main/scala/beam/router/skim/CsvSkimReader.scala

```diff
@@ -39,9 +39,11 @@ class CsvSkimReader(
             logger.warn(s"Could not load warmStart skim from '$aggregatedSkimsFilePath': ${ex.getMessage}")
             Map.empty[AbstractSkimmerKey, AbstractSkimmerInternal]
         }
-        .getOrElse(Map.empty[AbstractSkimmerKey, AbstractSkimmerInternal])
+        .getOrElse({
```

Comment:
Now it ends up with *really* duplicate code. Here is what I was suggesting earlier with trapping the main part in a variable

```
val skimMapTry = Try{
  IOUtils.getBufferedReader(aggregatedSkimsFilePath)
}.flatMap(tryReadSkims)
skimMapTry match {
  case Success(skimMap) => skimMap
  case Failure(ex) =>
    logger.warn(s"Could not load warmStart skim from '$aggregatedSkimsFilePath': ${ex.getMessage}")
    Map.empty[AbstractSkimmerKey, AbstractSkimmerInternal]
}
```

Reply:
thanks for snipppet . Applied this change